manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GnuWin32.Gperf
+PackageVersion: 3.0.1
+Installers:
+- InstallerUrl: https://sourceforge.net/projects/gnuwin32/files/gperf/3.0.1/gperf-3.0.1.exe/download
```

Comment:
```suggestion
- InstallerUrl: http://ftp.gnu.org/pub/gnu/gperf/gperf-3.0.1.tar.gz
```

Reply:
It's a source tarball. Not a installer. This suggestion doesn't work.